tests/functional/verification/issues/00120.vpr

```diff
@@ -0,0 +1,842 @@
+domain PyType {
```

Comment:
I guess this file is in here by accident?

Reply:
Yes, it shouldn't be there. I've just removed.